akd/src/local_auditing.rs

```diff
@@ -212,7 +212,7 @@ mod tests {
 
         let blob_name = AuditBlobName {
             current_hash: crate::hash::EMPTY_DIGEST,
-            previous_hash: [1u8; crate::hash::DIGEST_BYTES],
+            previous_hash: [1u8; crate::hash::EMPTY_DIGEST.len()],
```

Comment:
nit: why not use the `DIGEST_BYTES`? I see you removed the import, but you could still read it from `akd_core`. It seems an odd change to now rely on the output of a fn which isn't a const to build an array.

Reply:
Ah, good point, I think this was something I failed to clean up.